benchmarking/benchmark.py

```diff
@@ -225,15 +226,19 @@ def get_times_e3fp(X: pd.DataFrame):
                 mol = PropertyMol(mol)
                 mol.SetProp("_SMILES", smiles)
 
-                # getting a molecule and the fingerprint
-                mol, values = conf_gen.generate_conformers(mol)
-                fps = fprints_from_mol(mol, fprint_params=fprint_params)
+                try:
+                    # getting a molecule and the fingerprint
+                    mol, values = conf_gen.generate_conformers(mol)
+                    fps = fprints_from_mol(mol, fprint_params=fprint_params)
 
-                # chose the fingerprint with the lowest energy
-                energies = values[2]
-                fp = fps[np.argmin(energies)].fold(1024)
+                    # chose the fingerprint with the lowest energy
+                    energies = values[2]
+                    fp = fps[np.argmin(energies)].fold(1024)
+
+                    X_seq.append(fp.to_vector())
+                except RuntimeError:
+                    X_seq.append(np.full(shape=1024, fill_value=-1))
```

Comment:
Do not use constant 1024, this is a constructor parameter

Reply:
Also, maybe return all zeros instead of all -1?

Reply:
But isn't this a better way to handle such cases? Vector of -1's is easy to find and, e.g., to be excluded later by the user, while full of zeros is not very apparent and may lead to hidden false information in the data.

Reply:
MAP4, MHFP and E3FP have handling like this (with vector of -1's)

Reply:
Ok